### PR TITLE
[hash]: Add GH DB schema

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

**HLD:** https://github.com/sonic-net/SONiC/pull/1101

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -421,7 +422,7 @@ namespace swss {
 
 /***** STATE DATABASE *****/
 
-#define STATE_SWITCH_CAPABILITY_TABLE_NAME          "SWITCH_CAPABILITY_TABLE"
+#define STATE_SWITCH_CAPABILITY_TABLE_NAME          "SWITCH_CAPABILITY"
```
Comment (prsunny) [inline common/schema.h:425]: Please revert this change and follow the existing convention of _TABLE

Comment (nazariig) [reply common/schema.h:425]: @prsunny let me clarify the need of this change:

Today in SONiC define `STATE_SWITCH_CAPABILITY_TABLE_NAME` (which supposed to be a reference point to the switch capabilities table) is currently not used. Instead, we have a lot of places with hardcoded  string `SWITCH_CAPABILITY`. The motivation of this change is to start using it for switch hash OA with the relevant table name.

Once this merged, we can raise a separate PR and replace all hardcoded values (python, C++, etc.) with the relevant named definition. And only then change the table name to be following the convention you mentioned.
Since this is a separate activity, i would prefer to avoid doing it as part of current PR.

Please let me know what do you think.

Comment (prsunny) [reply common/schema.h:425]: ok

**Review by prsunny — APPROVED** [review]

**Comment by StormLiangMS** [comment]
@nazariig  looks like new feature, pls wait for next branch cut.

@prsunny  for vis.
